internal/weather/display.go

```diff
@@ -45,10 +45,28 @@ const (
 
 var directionSymbols = [...]string{"↑", "↗", "→", "↘", "↓", "↙", "←", "↖"}
 
+
 func celsiusToFahrenheit(c float64) float64 {
 	return c*9/5 + 32
 }
 
+// dewPointCelsius calculates dew point in Celsius given temperature (C) and relative humidity (%)
+func dewPointCelsius(tempC float64, humidity int) float64 {
+	t := tempC
+	rh := float64(humidity)
+	a := 17.27
```

Comment:
What are `a` and `b`? I think they should be constants

Reply:
These are from Magnus Tetens formula. And they are often called just `a` and `b`. 

Where `a` is constant "Magnus Coefficient" and `b` is temperature constant. So we can call it so. Or leave it as is, they often appear in this way. What do you prefer?

Reply:
I feel like if it represents something, let’s give it a name, otherwise single letters are fine